vulnerabilities/models.py

```diff
@@ -188,13 +196,13 @@ def __str__(self):
 
 class PackageRelatedVulnerability(models.Model):
 
-    package = models.ForeignKey(
-        Package, on_delete=models.CASCADE, related_name="vulnerable_package"
-    )
+    package = models.ForeignKey(Package, on_delete=models.CASCADE, related_name="package")
     vulnerability = models.ForeignKey(Vulnerability, on_delete=models.CASCADE)
-    patched_package = models.ForeignKey(
-        Package, on_delete=models.CASCADE, null=True, blank=True, related_name="patched_package"
+    source = models.TextField(null=True)
+    confidence = models.PositiveIntegerField(
+        default=1, validators=[MinValueValidator(0), MaxValueValidator(MAX_CONFIDENCE)]
```

Comment:
If you have a default, please align this with the "Inference" data class... I also wonder if having a default here makes sense at all? And if a default is needed why 1? why not 0? or max? or 42?
Help text would be nice too.

Reply:
Done. It defaults to `MAX_CONFIDENCE` now, also there's a help text for `source`